src/components/book-card/z-book-card/index.tsx

```diff
@@ -0,0 +1,314 @@
+            fill="black"
+          />
+          <path
+            d="M8.22229 10.732V8.24543C8.22229 7.97165 8.51057 7.79449 8.75375 7.91689L11.2404 9.16019C11.5109 9.29548 11.5109 9.682 11.2404 9.81728L8.75375 11.0606C8.50896 11.183 8.22229 11.0058 8.22229 10.732Z"
+            fill="black"
+          />
+          <path
+            d="M4.35704 7.87826C5.42438 7.87826 6.28963 7.013 6.28963 5.94566C6.28963 4.87831 5.42438 4.01306 4.35704 4.01306C3.28969 4.01306 2.42444 4.87831 2.42444 5.94566C2.42444 7.013 3.28969 7.87826 4.35704 7.87826Z"
+            fill="black"
+          />
+          <path
+            d="M5.6454 5.62354H3.0686V6.26773H5.6454V5.62354Z"
+            fill="white"
+          />
+          <path
+            d="M4.67917 4.65723H4.03497V7.23402H4.67917V4.65723Z"
+            fill="white"
+          />
+        </g>
+        <defs>
+          <clipPath id="clip0_11996_224">
+            <rect
+              width="19.326"
+              height="19.326"
+              fill="white"
+              transform="translate(0.169922 0.469604)"
+            />
+          </clipPath>
+        </defs>
+      </svg>
+    );
+  }
+
+  private renderLandscape(): HTMLDivElement {
+    return (
+      <Fragment>
+        <div class="main-content">
+          {this.renderCover()}
+          <div class="card-info">
+            <div class="book-data">
+              <div class="authors-title-cta-section">
+                <div class="authors-title">
+                  <div class="authors body-4">{this.authors}</div>
+                  {this.renderOperaTitle()}
+                </div>
+                <slot name="cta"></slot>
+              </div>
+              <div class="isbn-tags-link-section">
+                <div class="isbn-tags-section">
+                  <div class="volume-title body-4">{this.volumeTitle}</div>
+                  <div class="isbn body-4-sb">{this.isbn}</div>
+                  <div class="tags">{this.renderTags()}</div>
+                </div>
+                <div class="link-section">
+                  <div class="catalog-link">
+                    {this.catalogUrl && (
+                      <a
+                        href={this.catalogUrl}
+                        class="z-link z-link-icon body-4-sb"
+                        onClick={() => this.emitCatalogClick()}
+                      >
+                        Scheda catalogo
+                        <z-icon
+                          name="arrow-quad-north-east"
+                          height={16}
+                          width={16}
+                          fill="color-primary01-icon"
+                        ></z-icon>
+                      </a>
+                    )}
+                  </div>
+                  {this.adoption && <div class="adoption-tag body-5-sb">adottato</div>}
+                </div>
+              </div>
+            </div>
+            <div class="ebook-container">
+              {this.ebookUrl && (
```

Comment:
Da figma leggo che la sezione relativa all'ebook deve essere "sostituibile in base alle esigenze di prodotto", è sufficiente il check sulla presenza dell'ebook url?

Reply:
Sì, puoi provare da Storybook, ad esempio. Se non passi il link, la sezione non viene mostrata.

Reply:
si, questo ok, ma da figma è indicato come contenuto slottato quindi il mio dubbio è: basta il controllo sulla presenza del link che fa comparire/scomparire la sezione o deve proprio essere possibile sostituirla con altro?

Reply:
Facciamo chiarezza su questa cosa e poi eventualmente lo gestiamo nella storia della card verticale, direi.

Reply:
ok